File: ClientDashboard_API/DTOs/ClientMetricsDto.cs
namespace ClientDashboard_API.DTOs
{
    public class ClientMetricsDto
    {
        public int BaseClients { get; set; }

        public int AcquiredClients { get; set; }

        public double AcquisitionPercentage { get; set; }

        public int ChurnedClients { get; set; }

        public double ChurnPercentage { get; set; }

        public int NetGrowth { get; set; }

        public double NetGrowthPercentage { get; set; }

        public int SessionsPerClient { get; set; }

        public int MonthlyClientSessions { get; set; }

    }
}

File: ClientDashboard_GUI/src/app/models/dtos/client-metrics-dto.ts
export interface ClientMetricsDto {
  baseClients: number;
  acquiredClients: number;
  acquisitionPercentage: number;
  churnedClients: number;
  churnPercentage: number;
  netGrowth: number;
  netGrowthPercentage: number;
  sessionsPerClient: number;
  monthlyClientSessions: number;
}